doc/api/stream.md

```diff
@@ -1273,8 +1273,8 @@ If the `decodeStrings` property is set in the constructor options, then
 indicate the character encoding of the string. This is to support
 implementations that have an optimized handling for certain string
 data encodings. If the `decodeStrings` property is explicitly set to `false`,
-the `encoding` argument can be safely ignored, and `chunk` will always be a
-`Buffer`.
+the `encoding` argument can be safely ignored, and `chunk` will remain the same
+type from the source.
```

Comment:
When decodeStrings is true, string is converted into Buffer here.
https://github.com/nodejs/node/blob/v6.9.3-proposal/lib/_stream_writable.js#L285-L288
Of course Buffer is still Buffer. So the chunk always be a Buffer.

When decodeStrings is false, there is no conversion before `_write()` received it.
So, string keeps string, not always be Buffer.